test/e2e/systemd_notify_test.go

```diff
@@ -0,0 +1,204 @@
+// +build !remoteclient
+
+package integration
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	. "github.com/containers/libpod/test/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman systemd-notify", func() {
+	var (
+		tempdir    string
+		err        error
+		podmanTest *PodmanTestIntegration
+	)
+
+	BeforeEach(func() {
+		SkipIfRootless()
```

Comment:
This really bothers me, and I'm afraid I spent far too much time today trying to track down the AVC, never getting too far due to the distraction game.

Anyway, I'd love to see this limitation removed but I'm 99% convinced that the problem is outside the scope of your PR. This is really nice work, thank you.

Reply:
If it makes you feel any better - the binary appears to work just fine rootless in all 3 modes. I tried the test rootless but something in my environment is messed up and complaining about a lock... I had just copied that from the systemd_test.